components/infobox/wikis/fighters/infobox_league_custom.lua

```diff
@@ -0,0 +1,196 @@
+---
+-- @Liquipedia
+-- wiki=fighters
+-- page=Module:Infobox/League/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Abbreviation = require('Module:Abbreviation')
+local Class = require('Module:Class')
+local Game = require('Module:Game')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector', {requireDevIfEnabled = true})
+local League = Lua.import('Module:Infobox/League', {requireDevIfEnabled = true})
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+local Title = Widgets.Title
+local Chronology = Widgets.Chronology
+
+local _args
+local _league
+
+local CustomLeague = Class.new()
+local CustomInjector = Class.new(Injector)
+
+function CustomLeague.run(frame)
+	local league = League(frame)
+	_league = league
+	_args = _league.args
+
+	-- Abbreviations
+	_args.abbreviation = _args.abbreviation or CustomLeague.getAbbrFromSeries(_args.series)
```

Comment:
doesn't the commons one already do that?
https://github.com/Liquipedia/Lua-Modules/blob/main/components/infobox/commons/infobox_league.lua#L57

Reply:
Yup